test/index.js

```diff
@@ -2003,3 +2058,42 @@ describe('def', function() {
   });
 
 });
+
+describe('test', function() {
+
```

Comment:
We still have the tests you added for many of the exported types. :)

This block is intended to test `$.test`, particularly in relation to:

  - concrete parameterized types such as `$.Array($.Number)`, for which `t.test(['a', 'b', 'c'])` has erroneously evaluated `true` since #48; and

  - polymorphic parameterized types such as `$.Array(a)`, for which `t.test([1, 2, '3'])` has always erroneously evaluated `true`.